js/angular-tablesort.js

```diff
@@ -196,6 +196,13 @@ tableSortModule.directive('tsWrapper', ['$parse', '$compile', function( $parse,
                     }
                 }
             }
+			
+			if($attrs.tsFilterFields){
+				var filterFields = $attrs.tsFilterFields.split(",").filter(function(item){return item && item.trim() !== ""});
```

Comment:
Could you break this up so the body of the filter function is on it's own line.  That would make it a bit easier to read.